blueprint/src/chapter/ch06automorphicrepresentations.tex

```diff
@@ -20,7 +20,7 @@ \section{Central simple algebras}
     \label{IsCentralSimple}
     \lean{IsCentralSimple}
     \leanok
-A \emph{central simple algebra} over a field $K$ is a $K$-algebra $D$ such that $K$ is the centre of $D$
+A \emph{central simple algebra} over a field $K$ is a nonzero $K$-algebra $D$ such that $K$ is the centre of $D$
```

Comment:
When writing this, I decided that K being a field and K being the centre of D implied that D was nonzero, but there's no harm clarifying this.